src/darwin/Framework/CHIP/MTRBaseDevice.h

```diff
@@ -26,6 +26,9 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+typedef NSDictionary<NSString *, id> * MTRDeviceResponseValueDictionary;
+typedef NSDictionary<NSString *, id> * MTRDeviceDataValueDictionary;
```

Comment:
Please let's not have this be public API for now...  There are some issues with having public typedefs (like promising that the names will stay stable, etc)... For now, we should just have the typedef in one of the project headers and then think about the public API aspect.

Reply:
Addressed in https://github.com/project-chip/connectedhomeip/pull/32778